File: tests/xcalablemp/global-view/util/C-Square/mpi_xmp.c
#include "mpi.h"
#include "xmp.h"
extern int ixmp_sub();

int main(int argc, char **argv)
{
  MPI_Init(&argc, &argv);
  xmp_init(MPI_COMM_WORLD);

  ixmp_sub();

  xmp_finalize();
  MPI_Finalize();
  return 0;
}

File: tests/xcalablemp/global-view/util/F/mpi_xmp.f90
program test
  include 'mpif.h'
  integer ierror
  
  call MPI_INIT(ierror)
  call xmp_init(MPI_COMM_WORLD)
  call hoge()
  call xmp_finalize()
  call MPI_FINALIZE(ierror)
  
end program test
